src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -0,0 +1,108 @@
+package seedu.address.logic.commands;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Removes a tag associated with an existing person in the address book.
+ */
+public class TagCommand extends Command {
+
+    public static final String COMMAND_WORD = "tag";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid.";
```

Comment:
I think can potentially import this from logic/Messages.java to avoid repetition

Reply:
True, I will make the changes thanks